payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncherContractV2.kt

```diff
@@ -0,0 +1,82 @@
+package com.stripe.android.googlepaylauncher
+
+import android.content.Context
+import android.content.Intent
+import android.os.Parcelable
+import androidx.activity.result.contract.ActivityResultContract
+import androidx.annotation.RestrictTo
+import androidx.core.os.bundleOf
+import com.stripe.android.core.injection.InjectorKey
+import com.stripe.android.model.PaymentMethod
+import kotlinx.parcelize.Parcelize
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class GooglePayPaymentMethodLauncherContractV2 :
```

Comment:
Sad that this is needed. Can we document these somehow to fix up in our next major revision?

Reply:
Yep. This and `PaymentSheetContract` are the two contracts that are unfortunately public, though [this pull request](https://github.com/stripe/stripe-android/pull/6583) deprecates the latter. Everything that’s deprecated can then be removed in the next major bump.